fail2ban/server/server.py

```diff
@@ -523,27 +522,19 @@ def getAction(self, name, value):
 	def setBanTime(self, name, value):
 		self.__jails[name].actions.setBanTime(value)
 	
-	def addAttemptIP(self, name, *args):
-		return self.__jails[name].filter.addAttempt(*args)
+	def addAttemptIP(self, name, ip, failures):
```

Comment:
I'm not sure we should change the API here (`*failures` can be supplied similarly from outside).

Reply:
at least this would be better:
```diff
- def addAttemptIP(self, name, ip, failures):
+ def addAttemptIP(self, name, ip, *args):
```